DragaliaAPI.Integration.Test/Features/GraphQL/GraphQlTest.cs

```diff
@@ -40,9 +40,9 @@ public async Task Query_PlayerCharacters_ReturnsExpectedResult()
         GraphQLResponse<Response> response = await this.GraphQlHttpClient.SendQueryAsync<Response>(
             new GraphQLRequest
             {
-                Query = """
+                Query = $$"""
                 query {
-                    player(viewerId: 1) {
+                    player(viewerId: {{ViewerId}}) {
```

Comment:
Yes I know graphql has variables but it was refusing to work and kept throwing "error occured" on mutations that were fine in Postman